model/BlockChain_test.go

```diff
@@ -190,7 +191,14 @@ func TestBlockChain_BestBlockHash(t *testing.T) {
 		t.Error(err)
 		return
 	}
-	t.Logf("hash : %v, height : %d\n", hash, height)
+
+	if !bytes.Equal(hash[:], testBlcokChain.LastBlock.Hash[:]) {
+		t.Error("The two slice should be eqaul")
```

Comment:
Error description information should be described in detail , including error methods and reasons for error